contributing-guides/style-guide.md

```diff
@@ -91,3 +92,27 @@ The example above does not use a serial comma, so this could mean one of two thi
 This can be resolved by inserting a comma before the "and" or "or" in the final element in the list.
 
 > Delete the Git branches, tags, and remotes.
+
+## Chinese-Specific Rules
+
+When Chinese words, Latin words and Arabic numerals are written in the same sentence, it takes more attention to copywriting.
+
+The following guidelines are applied to Chinese (zh) and traditional Chinese (zh_TW):
+
+1. Place one space before/after English words.  
+   For example, use `列出所有 docker 容器` rather than `列出所有docker容器`.
+2. Place one space before/after numbers.  
+   For example, use `宽度为 50 个字` rather than `宽度为50个字`.
+3. Place one space between numbers and units **except** degrees and percentages.  
+   For example, use `容量 50 MB` rather than `容量 50MB`.  
+   For instances of degree and percentage, use `50°C` and `50%` rather than `50 °C` and `50 %`.
+4. No additional spaces before/after full-width punctuations.  
+   For example, use `开启 shell，进入交互模式` rather than `开启 shell ，进入交互模式`
+5. Use full-width punctuations except for long Latin clauses.  
+   For example, use `嗨，你好。` rather than `嗨, 你好.`
```

Comment:
> I think the most important rule is kind of missing: When sholudn't you use full-width.
> 
> Basically, that would be something like
> 
> ```
> 1. Use half-width punctuation after Latin characters.
>    For example, use 等效于 `vim -R`. rather than 等效于 `vim -R`。
> ```

I'll combine it with this rule.

Reply:
`Use half-width punctuation after Latin characters.` conflicts with [this](https://github.com/tldr-pages/tldr/pull/6285#issuecomment-893526569)

Reply:
But that's what I basically did in #5240. Did I misunderstand something? I'm confused.